release/staging-build.yml

```diff
@@ -1,35 +1,245 @@
+  - identifier: etcdadm_controller
+    buildspec: buildspec.yml
+    env:
+      variables:
+       PROJECT_PATH: projects/mrajashree/etcdadm-controller
+       CLONE_URL: https://git-codecommit.us-west-2.amazonaws.com/v1/repos/mrajashree.etcdadm-controller
+  - identifier: cluster_api_provider_tinkerbell 
+    buildspec: buildspec.yml
+    env:
+      variables:
+       PROJECT_PATH: projects/tinkerbell/cluster-api-provider-tinkerbell 
+       CLONE_URL: https://git-codecommit.us-west-2.amazonaws.com/v1/repos/tinkerbell.cluster-api-provider-tinkerbell 
+  - identifier: boots 
+    buildspec: buildspec.yml
+    env:
+      variables:
+       PROJECT_PATH: projects/tinkerbell/boots 
+       CLONE_URL: https://git-codecommit.us-west-2.amazonaws.com/v1/repos/tinkerbell.boots 
+  - identifier: hegel
+    buildspec: buildspec.yml
+    env:
+      variables:
+       PROJECT_PATH: projects/tinkerbell/hegel 
+       CLONE_URL: https://git-codecommit.us-west-2.amazonaws.com/v1/repos/tinkerbell.hegel 
+  - identifier: hub 
+    buildspec: buildspec.yml
+    env:
+      variables:
+       PROJECT_PATH: projects/tinkerbell/hub 
+       CLONE_URL: https://git-codecommit.us-west-2.amazonaws.com/v1/repos/tinkerbell.hub 
+  - identifier: pbnj
+    buildspec: buildspec.yml
+    env:
+      variables:
+       PROJECT_PATH: projects/tinkerbell/pbnj 
+       CLONE_URL: https://git-codecommit.us-west-2.amazonaws.com/v1/repos/tinkerbell.pbnj 
+  - identifier: hook 
+    buildspec: buildspec.yml
+    env:
+      variables:
+       PROJECT_PATH: projects/tinkerbell/hook 
+       CLONE_URL: https://git-codecommit.us-west-2.amazonaws.com/v1/repos/tinkerbell.hook
+  - identifier: cfssl 
+    buildspec: buildspec.yml
+    env:
+      variables:
+       PROJECT_PATH: projects/cloudflare/cfssl
+       CLONE_URL: https://git-codecommit.us-west-2.amazonaws.com/v1/repos/cloudflare.cfssl
+  - identifier: distribution 
+    buildspec: buildspec.yml
+    env:
+      variables:
+       PROJECT_PATH: projects/distribution/distribution
+       CLONE_URL: https://git-codecommit.us-west-2.amazonaws.com/v1/repos/distribution.distribution
+  - identifier: cilium 
+    buildspec: buildspec.yml
+    env:
+      variables:
+       PROJECT_PATH: projects/cilium/cilium
+       CLONE_URL: https://git-codecommit.us-west-2.amazonaws.com/v1/repos/cilium.cilium
+  - identifier: cluster_api 
+    buildspec: buildspec.yml
+    env:
+      variables:
+       PROJECT_PATH: projects/kubernetes-sigs/cluster-api
+       CLONE_URL: https://git-codecommit.us-west-2.amazonaws.com/v1/repos/kubernetes-sigs.vsphere-csi-driver
+  - identifier: cluster_api_provider_aws
+    buildspec: buildspec.yml
+    env:
+      variables:
+       PROJECT_PATH: projects/kubernetes-sigs/cluster-api-provider-aws
+       CLONE_URL: https://git-codecommit.us-west-2.amazonaws.com/v1/repos/kubernetes-sigs.cluster-api-provider-aws
+  - identifier: kind
```

Comment:
This depends on etcdadm and cri-tools and local path provisioner, double check there aren't more.

Reply:
yup i had them on before, but i think I removed them during testing. Will add them back.